docs/troubleshooting/paddle-threadlocal-coredump.md

```
@@ -0,0 +1,71 @@
+
+---
+title: PaddlePaddle CoreDump Issue with thread_local Allocator on vGPU
```

markdownlint-cli2 (0.23.2)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)